documentation/src/main/resources/theme/quarkus-theme.yml

```diff
@@ -0,0 +1,314 @@
+# Hibernate theme based on the default theme
```

Comment:
Same here, how about importing this file from some external location?

Reply:
https://github.com/datastax/cassandra-quarkus/pull/9#discussion_r423068194